cmd/main.go

```diff
@@ -42,7 +42,7 @@ var version = "development"
 
 type P2PServer interface {
 	Start() error
-	Shutdown() error
```

Comment:
Isn't it a good practice to stay with error return when shutting down? Probably we can't do anything with that, but at least we have a message and we can possibly kill it manually.

Reply:
You can see that neither implementation returns any error - so... why stay here with an error?